content/en/blog/2018-08-27-helm-switch-dco.md

```diff
@@ -7,63 +7,74 @@ authorlink: "https://mattfarina.com"
+When Helm was part of the Kubernetes project it, like the rest of Kubernetes,
+used the [CNCF Contributor License Agreement
+(CLA)](https://github.com/cncf/cla). This served Helm well for years. But, most
+of the CNCF projects use a [Developers Certificate of Origin
+(DCO)](https://developercertificate.org/) instead of a CLA. The exceptions are
+Kubernetes and gRPC. Upon Helm becoming a CNCF project itself we were asked if
+we wanted to move Helm to a DCO. After some careful consideration and a little
+research, the Helm maintainers voted to move to a DCO.  <!--more-->
 ## What Does This Solve? Why Switch?
 
-Making a change like this should have a good reason and we have one. It is often easier to get started contributing under a DCO than a CLA.
+Making a change like this should have a good reason and we have one. It is often
+easier to get started contributing under a DCO than a CLA.
 
-When one is developing software for a company they need to have the company sign the Corporate CLA prior to submitting contributions. That means there is a step after the business decides to contribute where legal documents need to be signed and exchanged. Once this is done there are steps to associate people with those legal documents. All of this takes time. In some companies this process can take weeks or longer.
+When one is developing software for a company they need to have the company sign
+the Corporate CLA prior to submitting contributions. That means there is a step
+after the business decides to contribute where legal documents need to be signed
+and exchanged. Once this is done there are steps to associate people with those
+legal documents. All of this takes time. In some companies this process can take
+weeks or longer.
 
 We wanted to make it simpler to contribute.
 
 ## What Is A DCO?
 
-A DCO is lightweight way for a developer to certify that they wrote or otherwise have the right to submit code or documentation to a project. The way a developer does this is by adding a `Signed-off-by` line to a commit. When they do this they are agreeing to the DCO.
+A DCO is lightweight way for a developer to certify that they wrote or otherwise
+have the right to submit code or documentation to a project. The way a developer
+does this is by adding a `Signed-off-by` line to a commit. When they do this
+they are agreeing to the DCO.
 
-The full text of the DCO can be found at https://developercertificate.org. It reads:
+The full text of the DCO can be found at https://developercertificate.org. It
+reads:
 
-> Developer Certificate of Origin
-> Version 1.1
+> Developer Certificate of Origin Version 1.1
+>
+> Copyright (C) 2004, 2006 The Linux Foundation and its contributors. 1
+> Letterman Drive Suite D4700 San Francisco, CA, 94129
```

Comment:
Should we move this "1" down since it's part of an address?

Reply:
Basically, I think we want to keep the address format:
> Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
> 1 Letterman Drive
> Suite D4700
> San Francisco, CA, 94129

Reply:
good call. This was a blanket "word wrap to 80 characters" script so I'm sure there may be other locations as well where the formatting is off.